src/components/date-picker/DatePicker.module.css

```
@@ -0,0 +1,278 @@
+.calendarWrapper {
+  background: var(--surface-secondary);
+  border: 2px solid var(--border-strong);
+  border-radius: 16px;
+  box-shadow: var(--shadow-xl);
+  overflow: hidden;
+}
+
+/* ===============================
+   Animations
+   =============================== */
+@keyframes fadeInScale {
```

Stylelint (17.3.0)

[error] Expected keyframe name "fadeInScale" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)